test/ngModule/deferInitialRender.spec.ts

```diff
@@ -6,7 +6,6 @@ import {
   APP_INITIALIZER,
   ApplicationInitStatus,
   Component,
-  NgModuleFactoryLoader,
   SystemJsNgModuleLoader,
```

Comment:
SystemJsNgModuleLoader should be removed as well in Angular 13, its only usage for the use lazy loading by string, which is removed here.

Reply:
Done.